index.html

```diff
@@ -1107,8 +1107,35 @@
 
 ## Principles for Notifications and Interruptions
 
-<aside class="issue">Connect intrusiveness to [=autonomy=], and write some principles.</aside>
+<aside class="issue">This is related to [=autonomy=].</aside>
 
+Notifications and other interruptive UI can be a powerful way to capture attention. 
+Depending on the operating system in use, a notification can appear outside of the
+browser context (for example, in a general notifications tray) or even cause a device to
+buzz or play an alert tone.
+Like all powerful features, notifications can be misused and can become an annoyance 
+or even used to manipulate behaviour and thus reduce [=autonomy=].
+
+<div class="practice">
+
+  <span class="practicelab" id="principle-notifications control">A [=user agent=] should help
+    users control notifications and other interruptive UI that can be used to manipulate behavior.</span>
+
+  [=User agents=] should provide UI that allows their users to audit which web [=sites=] have been granted 
+  permission to display alerts and to revoke these permissions.  
+  [=User agents=] should also apply some quality metric to the initial request for permissions to receive
+  notifications (for example, disallowing sites from requesting permission on first visit).
+  
+  <span class="practicelab" id="principle-notifications-context">Web [=sites=] should use notifications
+    only for information that their users have specifically requested.
+  </span>
+  
+  Web [=sites=] should tell their users what specific kind of information people can expect to 
+  receive, and how notifications can be turned off,  when requesting permission to send interruptive 
+  notifications.  Web [=sites=] should not request permission to send notifications when the user is unlikely to have sufficient knowledge (e.g. information about what kinds of notifications they are signing up for) to make an informed response. If it's unlikely that such information could have been provided then the [=user agent=] should apply mitigations (for example, warming about potential malicious use of the notifications API).
+  Permissions should be requested in [=context=].
```

Comment:
I think this is a different meaning of "context" than the one we've `<dfn>`ed in this document.

```suggestion
  Permissions should be requested in context.
```